browser/src/components/core/AutoGrowInput.tsx

```diff
@@ -45,18 +46,25 @@ export function AutoGrowInput({
     //   >
     //     {value}
     //   </span>
-    <div className={`grow-wrap ${className ? className : ""}`}>
-      {/* <div className={`grow-wrap`}> */}
-      <textarea
-        {...(extraProps || {})}
-        className="form-textarea mt-1 block w-full"
-        style={{ width: "300px", minHeight: "200px" }}
-        // placeholder="Enter your response to the prompt..."
-        value={value}
-        onChange={(event) => onChange(event.target.value)}
-      ></textarea>
-    </div>
-
+    <>
+      <div className={`grow-wrap ${className ? className : ""}`}>
+        {/* <div className={`grow-wrap`}>
+        <textarea
+          {...(extraProps || {})}
+          className="form-textarea mt-1 block w-full"
+          style={{ width: "300px", minHeight: "200px" }}
+          // placeholder="Enter your response to the prompt..."
+          // value={value}
+          // onChange={(event) => onChange(event.target.value)}
+        >
+        </textarea> */}
+        <Editor
+          value={value} 
```

Comment:
is this value pure HTML? and is it possible to serialize to markdown before we pass it to the database? would be good not to store raw HTML in the db

Reply:
will be working this out tomorrow!

Reply:
tiptap [doesn't support markdown output out the box](https://tiptap.dev/guide/output#not-an-option-markdown), but looks like we can use a ProseMirror (which tiptap is a wrapper for) [extension](https://prosemirror.net/examples/markdown/) to serialize to/from Markdown, will try this out.